### PR TITLE
make sound on quad drop and destroy

### PR DESCRIPTION
**Pull request by willharrison**

### CODE REVIEW

**Comment by ALSchwalm** [comment]
What is the source and license for the sound asset? Also is there any chance you could find something 'softer'. I'm thinking like the drop sound from [here](tetrisfriends.com/games/Battle2P/game.php)

**Comment by willharrison** [comment]
I got it from http://www.freesound.org/.  I will look and see what else I can find.  I was going to also use some type of drop sound for when the blocks are destroyed.

**Comment by willharrison** [comment]
I think these two sounds work well together.

**Comment by ALSchwalm** [comment]
Please do not commit sounds/.DS_Store. Feel free to add it to the git ignore. Also, it looks like some of the sounds from that site require attribution. See their faq [here](http://www.freesound.org/help/faq/#licenses). If the ones you selected need attribution, you can go ahead an add a section to the readme for that

**Comment by willharrison** [comment]
My mistake not adding DS_Store to the git ignore.  I will update the readme.

File: js/app/block.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ define(["app/config", "app/grid"], function(config, grid){
      * a square of blocks of the same color, do nothing. Otherwise, clear all
      * adjacent blocks (not counting diagonals) recursively.
      */
-    Block.prototype.clear = function() {
+    Block.prototype.clear = function(game) {
```
Comment (ALSchwalm) [inline js/app/block.js]: I don't see any reason for this change

File: js/app/state/preload.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * A module returning a function which will be executed to load game assets
  * @module app/state/preload
  */
-define(function(){
+define(['app/game'], function(game){
```
Comment (ALSchwalm) [inline js/app/state/preload.js]: Preload should not require 'game'. It is already passed to the preload callback (see the note [here](https://github.com/ALSchwalm/Quad/blob/master/js/app/game.js#L3))

**Comment by ALSchwalm** [comment]
lgtm, merging. Thanks

**Pull request merged**